include/staff/ticket-view.inc.php

```diff
@@ -751,7 +751,7 @@ class="icon-building icon-fixed-width"></i> <?php
     <li><a href="#relations" id="ticket-relations-tab"
         data-url="<?php
         echo sprintf('#tickets/%d/relations', $ticket->getId()); ?>"
-        ><?php echo __('Related Tickets');
+        ><?php echo __('Linked Tickets');
```

Comment:
This may not be true. This may actually be Related Tickets. We will do a further review on this one.